modules/xfeatures2d/src/daisy.cpp

```diff
@@ -53,6 +53,19 @@
 #include <fstream>
 #include <stdlib.h>
 
+#ifdef _MSC_VER
+    #if (_MSC_VER <= 1700)
+        /* This function rounds x to the nearest integer, but rounds halfway cases away from zero. */
+        static inline double round(double x)
```

Comment:
Yup, totally my fault, good catch, this was already fixed [here](https://github.com/patricksnape/opencv_contrib/commit/8247a7a0f59ba6c1fcfbac6a660db3f8acdc9ff8)